rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -1401,46 +1401,50 @@ function buildComplianceBar(compliance, minPxSize) {
 
   if(notApplicable != 0) {
     var value = compliancePercentValue(notApplicable);
-    content.append('<div class="progress-bar progress-bar-notapplicable" style=" width:'+widthArr[0]+'" title="Not applicable: '+notApplicable+'%">'+value+'%</div>');
+    var tooltip = formatOn2Digits(notApplicable);
+    content.append('<div class="progress-bar progress-bar-notapplicable" style=" width:'+widthArr[0]+'" title="Not applicable: '+tooltip+'%">'+value+'%</div>');
   }
 
   if(okStatus != 0) {
     var text = []
     if (success != 0) {
-      text.push("Success: "+success+"%");
+      text.push("Success: "+formatOn2Digits(success)+"%");
```

Comment:
I believe formatOn2Digits only keep the last two digits on a number, so will not work for "100" or floating point number (10.54 => 54)